lib/luaT/luaT.c

```diff
@@ -471,16 +471,15 @@ void luaT_registeratname(lua_State *L, const struct luaL_Reg *methods, const cha
 }
 
 
-/* utility functions */
-const char *luaT_classrootname(const char *tname)
+const char* luaT_classrootname(const char *tname)
```

Comment:
It's single use within this file but it's also a public function that can be used externally. So I think we should keep it, but maybe change its behaviour? It seems more natural to return `a.b` as the module name for a class `a.b.c` doesn't it? So I'd change the code of the function to do that and update the documentation accordingly.